TeslaSUI/TeslaSUI/Views/ClimateView.swift

```diff
@@ -0,0 +1,393 @@
+// ClimateView.swift
+// Copyright © RoadMap. All rights reserved.
+
+import SwiftUI
+
+/// Экран настроек климат контроля
+struct ClimateView: View {
+    // MARK: - Public Propeties
+    
+    var body: some View {
+        backgroundStackView {
+            ZStack {
+                VStack {
+                    HStack {
+                        backButtonView
+                        Spacer()
+                        screenNameTextView
+                        Spacer()
+                        settingsButtonView
+                    }
+                    ZStack {
+                        progressView
+                        trimCirlceProgress
+                        if climateViewModel.isAlertShown {
+                            alertView
+                        } else {
+                            EmptyView()
+                        }
+                    }
+                    Spacer()
+                    climateSettingsView
+                    Spacer()
+                }
+                BottomSheetView {
+                    VStack {
+                        bottomSheetCapsuleView
+                        Spacer()
+                        HStack {
+                            bottomSheetTextView
+                            Spacer()
+                            acIsOnButtonView
+                        }
+                        HStack(spacing: -20) {
+                            colorPickerView
+                            stepperView
+                            ventButtonView
+                        }
+                        .padding(.top)
+                        Spacer(minLength: 240)
```

Comment:
```suggestion
                BottomSheetView {
                    VStack {
                        bottomSheetCapsuleView
                        Spacer()
                        HStack {
                            bottomSheetTextView
                            Spacer()
                            acIsOnButtonView
                        }
                        HStack(spacing: -20) {
                            colorPickerView
                            stepperView
                            ventButtonView
                        }
                        .padding(.top)
                        Spacer(minLength: 240)
```
вынести в проперти, а то слишком много всего в боди
а в идеале боттомщит вынести тоже в отдельный файл, так как тут слишком много кода получается